### PR TITLE
Expand on Snowfork content on Bridge Hub page

### PR DESCRIPTION
**Pull request by CrackTheCode016**
Addresses mega issue #6085 

**Expand on Snowfork content on Bridge Hub page**

### CODE REVIEW

**Comment by CrackTheCode016** [comment]
Left it pretty simple, mostly pointing towards Snowbridge's documentation and summarizing its features.

File: docs/learn/learn-bridge-hub.md [hunk 1]
```diff
@@ -25,6 +29,11 @@ A two-way trustless bridge between chains A and B can be viewed as two one-way b
 source and a target chain. Any bridge operating on the Bridge Hub will have on-chain (pallets) and
 offchain (relayers) components.
 
+Examples of trustless on Bridge Hub include:
```
Comment (filippoweb3) [inline docs/learn/learn-bridge-hub.md]:
```suggestion
Examples of trustless bridges on Bridge Hub include:
```

File: docs/learn/learn-bridge-hub.md [hunk 2]
```diff
@@ -25,6 +29,11 @@ A two-way trustless bridge between chains A and B can be viewed as two one-way b
...
+
+- Snowfork, a decentralized bridge between Polkadot and Ethereum
```
Comment (filippoweb3) [inline docs/learn/learn-bridge-hub.md:34]: should we specify that this is an Ethereum light client on Polkadot that syncs to Ethereum and keeps the Ethereum state up-to-date on Polkadot?

**Review by filippoweb3 — APPROVED** [review]
Thanks for this PR. I think we can add the page to the Wiki although the snowbridge page can benefit from additional information about how the Ethereum state is kept up-to-date on Polkadot. This can be done on a separate PR later on :)

File: docs/learn/learn-bridge-hub.md [hunk 3]
```diff
@@ -2,20 +2,24 @@
...
-resource intensive can benefit from operating seaparately on system parachains. That's why, the
-Bridge Hub system parachain is operating on
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} since 2023. The Bridge Hub has all
-the required bridge pallets in its runtime, which enable trustless bridging with other blockchain
-networks like {{ polkadot: Kusama :polkadot }}{{ kusama: Polkadot :kusama }}, Ethereum etc. The
-Bridge Hub uses the native token of the relay chain,
+The primary functionality of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
+relay chain is to secure the parachains and facilitate secure communication between them. All other
+functionalities like asset transfers, governance, identities and bridging (a potentially resource
+intensive task) can benefit from operating separately on system parachains. System parachains are
```
Comment (profradha) [inline docs/learn/learn-bridge-hub.md]:
```suggestion
intensive task) can benefit from operating separately on [system parachains](./learn-system-chains.md). System parachains are
```

File: docs/learn/learn-bridge-hub.md [hunk 4]
```diff
@@ -2,20 +2,24 @@
...
+responsible for delegating functionality away from the relay chain for peformance reasons, taking
+advantage of the inherent parallelization the relay chain provides.
```
Comment (profradha) [inline docs/learn/learn-bridge-hub.md]:
```suggestion
advantage of the inherent parallelization the architecture of Polkadot provides.
```

File: docs/learn/learn-bridge-hub.md [hunk 5]
```diff
@@ -2,20 +2,24 @@
...
+
+The Bridge Hub system parachain operates on
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, and is responsible for faciliating
+bridges to the wider Web3 space. It contains all required bridge
```
Comment (profradha) [inline docs/learn/learn-bridge-hub.md]:
```suggestion
bridges to the wider Web3 space. It contains the required bridge
```

File: docs/learn/learn-bridge-hub.md [hunk 6]
```diff
@@ -2,20 +2,24 @@
...
+[pallets](../general/glossary.md#pallet) in its runtime, which enable trustless bridging with other
+blockchain networks like {{ polkadot: Kusama :polkadot }}{{ kusama: Polkadot :kusama }}, Ethereum
+etc. The Bridge Hub uses the native token of the relay chain,
```
Comment (profradha) [inline docs/learn/learn-bridge-hub.md]:
```suggestion
blockchain networks like {{ polkadot: Kusama :polkadot }}{{ kusama: Polkadot :kusama }} and Ethereum
. The Bridge Hub uses the native token of the relay chain,
```

File: docs/learn/learn-snowbridge.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+id: learn-snowbridge
+title: Snowbridge
+sidebar_label: Snowbridge
+description: Overview of Snowbridge by Snowfork
+keywords: [Bridge, XCM, Bridge Hub, Snowbridge, Snowfork]
+slug: ../learn-snowbridge
+---
+
+Snowbridge by Snowfork is a general-purpose, trustless bridge between Polkadot and Ethereum. It
```
Comment (profradha) [inline docs/learn/learn-snowbridge.md]:
```suggestion
Snowbridge by [Snowfork](https://snowfork.com/) is a general-purpose, trustless bridge between Polkadot and Ethereum. It
```

File: docs/learn/learn-snowbridge.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+utilizes the [Bridge Hub system parachain](./learn-bridge-hub.md) to establish a connection to its
+relayers, allowing for permissionless messaging between Ethereum and Polkadot.
+
+## Trustlessness in Snowbridge
+
+Trustlessness here is defined in the sense that the users do not need to trust any particular
+individuals or organizations, but rather only the mathematics, code, cryptography, and protocol. An
+example of a system which implies a high level of trust **trust** would be that of a bridge which is
```
Comment (profradha) [inline docs/learn/learn-snowbridge.md:18]:
```suggestion
example of a system which implies a high level of **trust** would be that of a bridge, which is
```

File: docs/learn/learn-snowbridge.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+## Applications
+
+Snowbridge currently supports
+[two-way token transfers](https://docs.snowbridge.network/applications/token-transfers) between
+Ethereum and Polkadot parachain.
+
+## Accessing & Using Snowbridge
+
+Snowbridge can be accessed through [the web app.](https://app.snowbridge.network/), where you may
```
Comment (profradha) [inline docs/learn/learn-snowbridge.md]:
```suggestion
Snowbridge can be accessed through [the web app](https://app.snowbridge.network/), where you may
```

File: docs/learn/learn-snowbridge.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+track [how long transfers are taking,](https://app.snowbridge.network/status),
```
Comment (profradha) [inline docs/learn/learn-snowbridge.md]:
```suggestion
track [processing times of transactions,](https://app.snowbridge.network/status),
```

**Review by DrW3RK — APPROVED** [review]

**Comment by CrackTheCode016** [comment]
@filippoweb3
> although the snowbridge page can benefit from additional information about how the Ethereum state is kept up-to-date on Polkadot.

Good point, I was not sure whether I should go in more detail about the message passing and its architecture, I agree we can add in a separate PR.

File: docs/learn/learn-bridge-hub.md (hunk 2 above)
Comment (CrackTheCode016) [reply docs/learn/learn-bridge-hub.md:34]: Maybe this is good for the next PR, we can add those details in the followup 👍

**Pull request merged**